root/frontend/src/routes/Home.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import MinimalHeader from '../components/MinimalHeader/MinimalHeader';
+
+function Home() {
+  return (
+    <div>
+      <MinimalHeader />
```

Comment:
I think we'd probably want the header to always be present. Would it make more sense to have the component rendered somewhere higher up rather than to be re-rendered per page?